### ISSUE TITLE
Panning in PDFs while zoomed in does not work

### ISSUE THREAD

**Issue by dead10ck** [foobnix/LibreraReader#634]
When you open a PDF document and zoom in, you cannot pan around the page. Dragging to the left or right always turns to the next page. Instead, panning to the side should always pan unless the screen is already at the edge of the document. This is with version 8.3.100-fdroid. Here is a video of the behavior:

https://vimeo.com/452899674

**Comment by Saumen95** [comment]
I faced this problem too.this happens only in book mode

for a personal hack,zoom your page to a degree that you can see the full page nice and easy.then use the lock button from below to fix that zoom level

**Comment by Ernest1338** [comment]
Yes, this is so frustrating if you need to zoom at some part of the page (small diagram for example).

**Comment by moshiur-raj** [comment]
Facing the same issue with the latest version in fdroid. Please fix this.

**Comment by wbrtm** [comment]
Using two fingers to pan is a workaround. Not perfect but works.
